packages/figma/src/lib/colors.ts

```diff
@@ -0,0 +1,24 @@
+export type webRGBA = [number, number, number, number]
+
+const defaultWebRgb = [0, 0, 0]
+
+export const cssRGBtoWebRgb = (rgb: string) => {
+  const match = rgb.match(/rgba?\((.*?)\)/)
+  const values = match && match[1].trim()
+  if (values) {
+    const sep = values.includes(',') ? ',' : ' '
+    const parts = values.split(sep)
+    const result = []
+    result.push(Number(parts[0]))
+    result.push(Number(parts[1]))
+    result.push(Number(parts[2]))
+    if (parts[3]) {
+      result.push(Number(parts[3]))
+    }
+    return result as webRGBA
+  }
+  return defaultWebRgb as webRGBA
+}
+export const isRgb = (value: string) => value.startsWith('rgb(') || value.startsWith('rgba(')
```

Comment:
А чего тут регулярку не стал писать?

```suggestion
export const isRgb = (value: string) => value.match(/^rgba?/)
```

Reply:
Просто понятнее, на мой вкус

Reply:
Но ведь у тебя ниже проверка на хекс через регулярку написана :) (насчет быстрее сомневаюсь, т.к. метод в v8 реализован через создание новой строки путем модификации и последующем сравнении)

Reply:
ладно пусть будет регулярка)